server/controllers/index.js

```diff
@@ -22,14 +29,16 @@ router.get('/auth', (req, res) => {
 /* Log in the user */
 router.post('/auth/login', (req, res) => {
     const { username } = req.body;
-    if (currentUsers.find((user) => user.username === username)) {
-        res.status(409).json({message: "Username already exists."});
-    } else {
-        req.session.user = username;
+    db.createUser(username)
+    .then(users => {
+        const id = users[0].dataValues.id;
```

Comment:
I think in general a create would return the new user or the id of the new user? Not `users`

Reply:
I think findOrCreate returns a list because it checks if a user with that username already exists and if it does, it returns that